src/System.Private.ServiceModel/tools/PRService/web.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
```

Comment:
I don't think your code has any dependency on 4.6 particularly (it may even work on 2.0). We potentially can skip web.config if you like smaller footprint :smile:. Your call.